canonical-websites/www.ubuntu.com
# Rebase Takeovers Branch with Master

## Done

Rebased Takeovers with Master to get Engagement Page link commits by David Calle.
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Click the "Read More" Button on the Juju Takeover should redirect to /engage/evolving-software?utm_source=takeover&utm_medium=FY19_Cloud_Juju_Whitepaper

## Screenshots
![image](https://user-images.githubusercontent.com/43535482/47367282-4a708180-d6d7-11e8-9afa-0dd7a6867e50.png)
